### DISCUSSION TITLE
Old milestone still growing?

### DISCUSSION THREAD

**Discussion by atc0005** [goreleaser/nfpm#711] [General]
@caarlos0 

The v2.32.0 milestone is the latest stable release per https://github.com/goreleaser/nfpm/releases.

I notice that the v2.27.0 release milestone (https://github.com/goreleaser/nfpm/milestone/17) is still open and new PRs are still being assigned to it. Is this intentional?

**Comment by caarlos0** [comment]
ah, good catch, not really using milestones here anymore, closed that one.

thanks for letting me know 🙏🏻

**Reply by caarlos0** [reply]
also just created the missing milestones
